### ISSUE TITLE
Munin is extremely resource hungry

### ISSUE THREAD

**Issue by imincik** [imincik/gis-lab#448]
Munin is extremely resource hungry when running 'munin-update' process. It is capable to eat whole RAM when running during the installation via Vagrant provisioner (1GB of RAM).
Maybe it can be related to fact, that all machines which it is trying to connect are down.

**Comment by imincik** [comment]
We can change munin command running from cron to execute something like:

```
/usr/bin/munin-cron --host server $(serf members -tag role=client -status=alive | awk -F " " '{printf "--host %s.gis.lab ", $1}')
```

**Comment by mpastor** [comment]
Fixed in 1256ff1314.

**Comment by imincik** [comment]
I am proposing to place our custom Munin script to '/opt/gislab/system/stats/update-stats.sh'. Reopening this issue for this purpose.
